src/tokens/lst_tokens/token_lst_clear.c

```diff
@@ -6,12 +6,17 @@
 /*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/03/08 19:32:11 by mcourtoi          #+#    #+#             */
-/*   Updated: 2023/03/09 14:59:23 by mcourtoi         ###   ########.fr       */
+/*   Updated: 2023/04/03 17:45:52 by mcourtoi         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
+/**
+ * @brief Clear each node of the list with token_lst_del_one.
```

Comment:
The `brief` section is not supposed to explain how the function proceeds, but only what it does.